settings.py

```diff
@@ -242,6 +242,8 @@ def get_default_gateway_linux():
 PROD_DETAILS_CACHE_NAME = 'product_details'
 PROD_DETAILS_CACHE_TIMEOUT = None
 
+RECOVER_MSG_LANGS = config('RECOVER_MSG_LANGS', 'en', cast=Csv())
```

Comment:
for `decouple` you're supposed to provide it with a default in the form it will see in the environment unless the `cast` function can handle other types. You can see above that we provide the list of `ALLOWED_HOSTS` with a string.